server/api/impl/admin/synchronize.go

```diff
@@ -137,7 +137,12 @@ func (adm *Admin) LoadAppsList() error {
 	return nil
 }
 
-func (adm *Admin) UninstallApp(app *apps.App) error {
+func (adm *Admin) UninstallApp(appID apps.AppID) error {
+	app, err := adm.store.App().Get(appID)
```

Comment:
To add (admin) permission check?

Reply:
@catalintomai I've added a permission check check in 94e7ab16c6927ca8124b5de876d7a0a62552cc34. Please let me know what you think about that implementation.